pkg/receive/handler.go

```diff
@@ -810,3 +828,42 @@ func (p *peerGroup) get(ctx context.Context, addr string) (storepb.WriteableStor
 	p.cache[addr] = client
 	return client, nil
 }
+
+// GetTenantFromCertificate extracts the tenant value from a client's presented certificate. The x509 field to use as
```

Comment:
nit: To match method name and comment
```suggestion
// getTenantFromCertificate extracts the tenant value from a client's presented certificate. The x509 field to use as
```